inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/enums/ClusterType.java

```diff
@@ -27,4 +30,24 @@ public class ClusterType {
     public static final String PULSAR = "PULSAR";
     public static final String DATAPROXY = "DATAPROXY";
     public static final String KAFKA = "KAFKA";
+
+    private static final Set<String> TYPE_SET = new HashSet<String>() {
+        {
+            add(ClusterType.AGENT);
+            add(ClusterType.TUBEMQ);
+            add(ClusterType.PULSAR);
+            add(ClusterType.DATAPROXY);
+            add(ClusterType.KAFKA);
+        }
+    };
+
+    public static String forType(String clusterType) {
+        for (String type : TYPE_SET) {
```

Comment:
Why use for loop on a Set? Suggest getting directly.

Reply:
Done.